src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -378,6 +378,11 @@ public static int GetArity(this ISymbol symbol)
         [return: NotNullIfNotNull(parameterName: "symbol")]
         public static ISymbol? GetOriginalUnreducedDefinition(this ISymbol? symbol)
         {
+            if (symbol.IsTupleField())
+            {
+                return symbol;
```

Comment:
is this really waht we want?  not symbol.OriginalDefinition?  Could this leave instantated generic types in here?

Reply:
I think this is correct. In the past, tuples and their fields were definitions. I'm emulating this behavior here even though the underlying compiler API changed.